Deprecation of Legacy-UIComponents-Service

kergomard opened the pull request:
The Technical Board proposes to advance the move to the UI-Framework. This PR defines a way to get rid of Services/UIComponents with ILIAS 10 and to remove already deprecated functions with ILIAS 9, thus simplifying our code base and the work needed to keep a consistent appearance for our end users.

Commit:
Deprecation of Legacy-UIComponents-Service

matthiaskunkel commented:
Jour Fixe, 21 FEB 2022 : We highly appreciate this suggestion to push the UI Kitchen Sink and get rid of a concurrent UI component service. It would increase acceptance among maintainers if for all deprecated components an alternative KS implementation could already be offered.

kergomard committed:
Pushing Back Deadlines by one Version

Due to the delays releasing ILIAS 9 we decided to push this back one
ILIAS Version

kergomard commented:
We merge this now, as the branch of ILIAS8 has been created.
